pkg/models/iam/am.go

```diff
@@ -657,11 +669,6 @@ func CreateClusterRoleBinding(username string, clusterRoleName string) error {
 			glog.Errorln("delete cluster role binding", err)
 			return err
 		}
-		if found.RoleRef.Name == constants.ClusterAdmin {
```

Comment:
if found.RoleRef.Name != constants.ClusterAdmin => delete kubectl deploy?

Reply:
delete kubectl pod if cluster role is not cluster-admin, whether it exists or not